src/components/footer/footer.import.less

```diff
@@ -63,9 +63,8 @@
     font-size: 20px;
     text-align: center;
     vertical-align: auto;
-    line-height: 32px;
-    border: 2px solid #fff;
-    border-radius: 100%;
+    line-height: 32px; // border: 2px solid #fff;
+    // border-radius: 100%;
   }
```

Comment:
@nicholasmiller1 next time go ahead and remove the commented styles. If we ever need them again, they're saved in git